database/functions/R__application.find_site.sql

```diff
@@ -0,0 +1,79 @@
+--
+-- Name: find_site(uuid); Type: FUNCTION; Schema: application; Owner: postgres
+--
+
+CREATE FUNCTION application.find_site(
```

Comment:
We're going to need to do some refactoring of this and the find_sites DB function but we can do this under a separate ticket in the next sprint or something.

Reply:
Ok, sounds good to me.